.github/workflows/ci.yml

```diff
@@ -88,6 +85,26 @@ jobs:
           node-version: 18
       - run: npm --prefix ${{ github.workspace }}/internal/nginx/modules install-ci-test
 
+  release:
+    name: Release
+    runs-on: ubuntu-20.04
+    needs: [unit-tests, njs-unit-tests]
+    if: ${{ github.event_name == 'push' && ! github.ref != 'refs/heads/main' }}
```

Comment:
What is that sole exclamation for?

Also, should we prevent this from running on forks?

Reply:
oh wow that's a typo, good catch!

This is only running on push events, so it will be triggered in somebody's fork when they push a tag or a commit to the release branch, but it won't impact this repo, right? I don't think we need to block it?

Reply:
Looks like it would occur on any branch, right? And I guess it depends on what release steps are actually being run. Are images being pushed?

Reply:
it won't run on any branch, only the ones specified at the top on the push condition
```
on:
  push:
    branches:
      - main
      - release-*
    tags:
      - 'v[0-9]+.[0-9]+.[0-9]+*'
```
so just release branches and tags (since there's `github.ref != 'refs/heads/main'`).

This specific step is only about the release notes and publishing them to GitHub, the step for images is further down.
But for any of them it will run in their forks, for this step for example they will publish release notes on their fork.

Reply:
Oh ok, then it's probably fine.